### PR TITLE
Add LTS Table entry for LTS_07_2022_Ref01

### PR DESCRIPTION
**Pull request by vaavva**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -270,8 +270,8 @@ Below is a table showing the mapping of the LTS branches to the packages release
 
   | Package | GitHub Branch | LTS Tag | LTS Start Date | Maintenance End Date |
   | :-----: | :-----------: | :-----: | :------------: | :------------------: |
+  | vcpkg: 2022-07-18 | lts_07_2022 | LTS_07_2022_Ref01 | 2022-07-18 | 2023-07-18 |
   | vcpkg: 2022-01-21 | lts_01_2022 | LTS_01_2022_Ref01 | 2022-01-21 | 2023-01-21 |
-  | vcpkg: 2021-09-09 | lts_07_2021 | LTS_07_2021_Ref01 | 2021-08-11 | 2022-08-11 |
```
Comment (ewertons) [inline readme.md:274]: Technically we should not remove the entry unless it has passed the Maintenance End Date. Is the script doing that automatically? If so, I think that is not good. Only what was already passed the date should be removed.

**Review by ewertons — APPROVED** [review]

File: readme.md (hunk 1 above)
Comment (vaavva) [reply readme.md:274]: The script didn't do that automatically, I can revert it!

**Review by wduraes — APPROVED** [review]

**Comment by ewertons** [comment]
Not sure why horton is failing. I disabled horton and arduino checks and merged, since this is a change just in documentation.

**Comment by ericwol-msft** [comment]
@ewertons  Bert fixed the Horton issue.

**Pull request merged**